lib/tooling/base-tool.ts

```diff
@@ -46,7 +46,7 @@ const toolCounter = new PromClient.Counter({
 export class BaseTool {
     protected tool: ToolInfo;
     private env: ToolEnv;
-    private addOptionsToToolArgs = true;
+    protected addOptionsToToolArgs = true;
```

Comment:
my ide complained about this - many subtools change it but it's marked "private"